arraycontext/impl/jax/__init__.py

```diff
@@ -60,6 +59,23 @@ def _get_fake_numpy_namespace(self):
         from .fake_numpy import EagerJAXFakeNumpyNamespace
         return EagerJAXFakeNumpyNamespace(self)
 
+    def _map_container(self, func, array, allowed_types=None):
```

Comment:
Type annotation?

Reply:
Added in https://github.com/inducer/arraycontext/compare/f4ab671ce9eb96e9dd2397d9a77d5e7ac7c9be36..971df793aed6b5b107be09588c0791a2f664c2b9